packages/sdk/src/FlagResolverClient.ts

```diff
@@ -180,15 +191,53 @@ export class FetchingFlagResolverClient implements FlagResolverClient {
       const start = Date.now();
 
       return this.cacheReadThrough(context, () => this.resolveFlagsJson(request, signalWithTimeout))
-        .then(response => FlagResolution.ready(context, response, this.createApplier(response.resolveToken)))
+        .then(result => {
+          const latency = Date.now() - start;
+          if (result.isFromCache) {
+            this.traceConsumer({
+              requestTrace: {
+                millisecondDuration: latency,
+                status: LibraryTraces_Trace_RequestTrace_Status.STATUS_CACHED,
+              },
+            });
```

Comment:
A lot of repetition around these calls, maybe add a method like the old markLatency that just takes a latency and status?